### PR TITLE
Automatically deploy dist to dist branch

### PR DESCRIPTION
**Pull request by Jesus89**
Every commit in `public-api` (it could be changed) generates automatically the distribution files and @Cartofante commits the changes to the `dist` branch.

Dist branch: https://github.com/CartoDB/cartodb.js/tree/dist.
CDN: https://cdn.rawgit.com/CartoDB/cartodb.js/dist/public/carto.js

### CODE REVIEW

File: scripts/deploy-to-dist.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  exit 0
+fi
+
+echo "Starting deployment"
+echo "Target: dist branch"
+
+DIST_DIR="dist"
+TMP_DIST_DIR="tmp_dist"
+PUBLIC_DIR="public"
+INTERNAL_DIR="internal"
```
Comment (alonsogarciapablo) [inline scripts/deploy-to-dist.sh]: There is no need to "upload" the internal dist files to this branch IMO.

Comment (Jesus89) [reply scripts/deploy-to-dist.sh]: This can be useful for the Builder in the future

Comment (Jesus89) [reply scripts/deploy-to-dist.sh]: But I can remove it for now

**Comment by Jesus89** [comment]
I think we can use this now. I would leave the versions for the public (next) release

**Pull request merged**